cms/bundles/panels.py

```diff
@@ -17,34 +22,55 @@ def __init__(self, **kwargs: Any) -> None:
             self.content = self._content_for_instance(self.instance)
 
         def _content_for_instance(self, instance: "Model") -> Union[str, "SafeString"]:
-            if not hasattr(instance, "bundles"):
+            if not hasattr(instance, "active_bundle"):
                 return ""
 
-            if bundles := instance.bundles:
-                content_html = format_html_join(
-                    "\n",
-                    "<li>{} (Status: {})</li>",
-                    (
-                        (
-                            bundle.name,
-                            bundle.get_status_display(),
-                        )
-                        for bundle in bundles
-                    ),
-                )
-
-                content = format_html("<p>{}</p><ul>{}</ul>", "This page is in the following bundle(s):", content_html)
+            if bundle := instance.active_bundle:
+                if user_can_manage_bundles(self.request.user):
+                    content = format_html(
+                        "<p>This page is in the following bundle: "
+                        '<a href="{}" target="_blank" title="Manage bundle">{}</a> (Status: {})</p>',
+                        reverse("bundle:edit", args=[bundle.pk]),
+                        bundle.name,
+                        bundle.get_status_display(),
+                    )
+                else:
+                    content = format_html(
+                        "<p>This page is in the following bundle: {} (Status: {})</p>",
+                        bundle.name,
+                        bundle.get_status_display(),
+                    )
             else:
                 content = format_html("<p>{}</p>", "This page is not part of any bundles.")
             return content
 
 
-class PageChooserWithStatusPanel(PageChooserPanel):
+class CustomAdminPageChooser(PagesWithDraftsForBundleChooserWidget):
+    def get_display_title(self, instance: "Page") -> str:
+        title: str = instance.specific_deferred.get_admin_display_title()
+
+        if workflow_state := instance.current_workflow_state:
+            title = f"{title} ({workflow_state.current_task_state.task.name})"
+        else:
+            title = f"{title} (not in a workflow)"
+
+        return title
```

Comment:
Non-blocking nit-pick/suggestion to avoid repeating `title`. However one could argue your version is more readable.
```suggestion
        title: str = instance.specific_deferred.get_admin_display_title()

        if workflow_state := instance.current_workflow_state:
            title += f" ({workflow_state.current_task_state.task.name})"
        else:
            title += " (not in a workflow)"

        return title
```

Reply:
They amount to the same, but IMHO my version is more readable.

```python
       title: str = instance.specific_deferred.get_admin_display_title()

        if workflow_state := instance.current_workflow_state:
            return f"{title} ({workflow_state.current_task_state.task.name})"

        return f"{title} (not in a workflow)"
```

is better still. will amend

Reply:
a9fdade